MetaMask/metamask-extension
# [TS dashboard] Be consistent with terminology

## Explanation

There are two sides to the TypeScript dashboard code, the build scripts
and the app code. Currently we use the word "module" in the former but
"file" in the latter. Similarly, we use "partition" in the build code
but "level" in the app code. All of the logic that powers the
visualization you see on the dashboard is complicated enough, so there's
no reason to use duplicate terminology.

## Manual Testing Steps

Run `yarn ts-migration:dashboard:watch`, then open `development/ts-migration-dashboard/build/final/index.html`. You should be able to see the dashboard.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
